src/Apps/__tests__/Fixtures/Artist/Routes/ArticlesFixture.ts

```diff
@@ -1,212 +1,219 @@
+import { Articles_Test_QueryRawResponse } from "__generated__/Articles_Test_Query.graphql"
+
 export const ArticlesFixture = {
-  id: "pablo-picasso",
-  articlesConnection: {
-    pageInfo: { hasNextPage: true, endCursor: "YXJyYXljb25uZWN0aW9uOjk=" },
-    pageCursors: {
-      around: [
-        { cursor: "YXJyYXljb25uZWN0aW9uOi0x", page: 1, isCurrent: true },
-        { cursor: "YXJyYXljb25uZWN0aW9uOjk=", page: 2, isCurrent: false },
-        { cursor: "YXJyYXljb25uZWN0aW9uOjE5", page: 3, isCurrent: false },
-        { cursor: "YXJyYXljb25uZWN0aW9uOjI5", page: 4, isCurrent: false },
-      ],
-      first: null,
-      last: null,
-      previous: null,
-    },
-    edges: [
-      {
-        node: {
-          href: "/article/artsy-editorial-us-china-trade-war-art-market",
-          thumbnail_title:
-            "What the U.S.–China Trade War Means for the Art Market",
-          author: {
-            name: "Artsy Editorial",
-            __id: "QXV0aG9yOjVhYmJmYzZlY2I0YzI3MTczNzcyOWE1YQ==",
-          },
-          published_at: "Jul 23rd, 2018",
-          thumbnail_image: {
-            resized: {
-              url:
-                "https://d7hftxdivxxvm.cloudfront.net?resize_to=width&width=300&quality=80&src=https%3A%2F%2Fartsy-media-uploads.s3.amazonaws.com%2F0wcjPCKn9wQg_DRvywHDsQ%252FGettyImages-997234250-1200.jpg",
+  artist: {
+    slug: "pablo-picasso",
+    id: "blah",
+    articlesConnection: {
+      pageInfo: { hasNextPage: true, endCursor: "YXJyYXljb25uZWN0aW9uOjk=" },
+      pageCursors: {
+        around: [
+          { cursor: "YXJyYXljb25uZWN0aW9uOi0x", page: 1, isCurrent: true },
+          { cursor: "YXJyYXljb25uZWN0aW9uOjk=", page: 2, isCurrent: false },
+          { cursor: "YXJyYXljb25uZWN0aW9uOjE5", page: 3, isCurrent: false },
+          { cursor: "YXJyYXljb25uZWN0aW9uOjI5", page: 4, isCurrent: false },
+        ],
+        first: null,
+        last: null,
+        previous: null,
+      },
+      edges: [
+        {
+          node: {
+            href: "/article/artsy-editorial-us-china-trade-war-art-market",
+            thumbnail_title:
+              "What the U.S.–China Trade War Means for the Art Market",
+            author: {
+              name: "Artsy Editorial",
+              __id: "QXV0aG9yOjVhYmJmYzZlY2I0YzI3MTczNzcyOWE1YQ==",
```

Comment:
I suppose our mock network later is generating IDs for these fixtures, because this field no longer exists in the schema.